project/Build.scala

```diff
@@ -167,10 +167,12 @@ object ScalaMeterBuild extends MechaRepoBuild {
       "org.apache.commons" % "commons-math3" % "3.2",
       "org.scala-lang" % "scala-reflect" % "2.11.0",
       "org.scala-lang.modules" %% "scala-xml" % "1.0.1",
-      "org.scala-lang.modules" %% "scala-parser-combinators" % "1.0.1"
+      "org.scala-lang.modules" %% "scala-parser-combinators" % "1.0.1",
```

Comment:
Ouch, sorry for that. This is residue after first design (unpushed), where specific case classes were annotated to use specific deserializers instead of current design.